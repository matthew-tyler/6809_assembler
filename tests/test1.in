start:
    leay helloworld,pcr
    jsr print
    rts

const textscreenbase=$400

print:
    pshs a,x,y
    ldx #textscreenbase
printloop:
    lda ,y+
    beq printover
    cmpa #$40
    bhs print6847
    adda #$40
print6847:
    sta ,x+
    bra printloop
printover:
    puls a,x,y
    rts


helloworld:
    fcb "HELLO WORLD",0

