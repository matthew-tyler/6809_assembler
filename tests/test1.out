318D001ABD40083934328E0400A6A0270A814024028B40A78020F235323948454C4C4F20574F524C44